src/test/javascript/spec/component/shared/metis/postings-reactions-bar/posting-reactions-bar.component.spec.ts

```
@@ -271,16 +352,17 @@ describe('PostReactionsBarComponent', () => {
         { type: ConversationType.ONE_TO_ONE },
     ])('should initialize user authority and reactions correctly with same user', (dto: ConversationDTO) => {
-        component.posting!.author!.id = 99;
         jest.spyOn(metisService, 'getCurrentConversation').mockReturnValue(dto);
         jest.spyOn(accountService, 'userIdentity', 'get').mockReturnValue({ id: 99 } as User);
 
         reactionToDelete.user = { id: 99 } as User;
         post.reactions = [reactionToDelete];
-        component.posting = post;
-
+        runInInjectionContext(fixture.debugElement.injector, () => {
+            post.author!!.id = 99;
```

Biome (1.9.4)

[error] Forbidden extra non-null assertion.

Safe fix: Remove extra non-null assertion.


(lint/suspicious/noExtraNonNullAssertion)